spec/controllers/api/v1/orders_controller_spec.rb

```diff
@@ -75,6 +76,30 @@
         expect(parsed_body['designations'].count).to eq(0)
         expect(parsed_body['meta']['total_pages']).to eql(0)
       end
+
+      it 'returns goodcity order if search text is non draft goodcity order with toDesignateItem params' do
+        request.headers["X-GOODCITY-APP-NAME"] = "stock.goodcity"
```

Comment:
This line could be moved to before{} on each test. not an issue, just mentioning it :)